src/cli.cr

```diff
@@ -7,18 +7,21 @@ module Shards
       shards [<options>...] [<command>]
 
       Commands:
-          build [<targets>] [<options>]  - Build the specified <targets> in `bin` path.
-          check                          - Verify all dependencies are installed.
-          init                           - Initialize a `shard.yml` file.
-          install                        - Install dependencies, creating or using the `shard.lock` file.
-          list [--tree]                  - List installed dependencies.
-          lock [--update] [<shards>]     - Lock dependencies in `shard.lock` but doesn't install them.
-          outdated [--pre]               - List dependencies that are outdated.
-          prune                          - Remove unused dependencies from `lib` folder.
-          update [<shards>]              - Update dependencies and `shard.lock`.
-          version [<path>]               - Print the current version of the shard.
-
-      Options:
+          build [<targets>] [<build_options>]  - Build the specified <targets> in `bin` path.
+          check                                - Verify all dependencies are installed.
+          init                                 - Initialize a `shard.yml` file.
+          install                              - Install dependencies, creating or using the `shard.lock` file.
+          list [--tree]                        - List installed dependencies.
+          lock [--update] [<shards>]           - Lock dependencies in `shard.lock` but doesn't install them.
+          outdated [--pre]                     - List dependencies that are outdated.
+          prune                                - Remove unused dependencies from `lib` folder.
+          update [<shards>]                    - Update dependencies and `shard.lock`.
+          version [<path>]                     - Print the current version of the shard.
+      
+      Build options:
+          Use "crystal build --help" to see available build options.
```

Comment:
I don't think this should be mentioned here. There are no explanations for other command's arguments either.
That would better fit for a more detailled `shards build --help` if that was a thing.

Reply:
Since `shards build --help` is not a thing, I'm assuming we have to somehow clarify that `build_options` have nothing to do with `GENERAL OPTIONS`, same way it's stated in the manpage.

I also don't see a way how we could implement `shards build --help` without copying the info from `crystal build --help` directly and then somehow keeping these two in sync. 

Any ideas?

Reply:
Well `shards build --help` wouldn't list the options explicitly but show more details information about the command itself, just like the `build` entry on the manpage.

Reply:
Okay, we can specify it in the `build` arg itself, does this seem suitable?
```
build [<targets>] [<build_options>]  - Build the specified <targets> in `bin` path, all build_options are delegated to `crystal build`.
```

Reply:
Thanks @RX14! I've updated the code accordingly.